sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -214,6 +214,7 @@ export class EducationProgramControllerService {
       institutionProgramCode: program.institutionProgramCode,
       minHoursWeek: program.minHoursWeek,
       isAviationProgram: program.isAviationProgram,
+      credentialTypesAvi: program.credentialTypesAviation,
```

Comment:
for consistency can we follow the same key in API(form) similar to entity `credentialTypesAviation` ?